Dockerfile

```diff
@@ -1,12 +1,42 @@
-from ruby:3.2.0
-
-env \
-  DEBIAN_FRONTEND=noninteractive \
-  NODE_VERSION=19.0.0
-
-run \
-  sed -i "/deb-src/d" /etc/apt/sources.list && \
-  apt-get update && \
-  apt-get install -y build-essential libpq-dev postgresql-client unzip cmake && \
-  curl -sSL "https://nodejs.org/dist/v$NODE_VERSION/node-v$NODE_VERSION-linux-x64.tar.xz" | tar xfJ - -C /usr/local --strip-components=1 && \
-  npm install -g yarn
+from ruby:3.2.1-alpine
+
+RUN apk add \
+    # g++ is required for sassc
+    g++ \
+    postgresql-dev \
+    postgresql-client \
+    nodejs \
+    npm \
+    tzdata
+
+# for nokogiri
+RUN apk add --virtual .ruby-gemdeps libc-dev gcc libxml2-dev libxslt-dev make libffi-dev
+
+RUN npm install -g yarn
```

Comment:
Something like this:

```suggestion
# apk dependencies:
# sassc: g++
# nokogiri: .ruby-gemdeps libc-dev gcc libxml2-dev libxslt-dev make libffi-dev
RUN apk add \
    g++ postgresql-dev postgresql-client nodejs npm \
    tzdata .ruby-gemdeps libc-dev gcc libxml2-dev libxslt-dev make libffi-dev &&
     npm install -g yarn
```

:v:

Totally up to you :v:, there is no need to be too restrict with the number of layers.

Reply:
Does this work even without the `--virtual`?

Reply:
> Does this work even without the `--virtual`?

I actually don't know, I am not used to alpine, my bad.

Something like this should 100% work:

```suggestion
# apk dependencies:
# sassc: g++
# nokogiri: .ruby-gemdeps libc-dev gcc libxml2-dev libxslt-dev make libffi-dev
RUN apk add \
    g++ postgresql-dev postgresql-client nodejs npm tzdata && \
    apk add --virtual .ruby-gemdeps libc-dev gcc libxml2-dev libxslt-dev make libffi-dev && \
    npm install -g yarn
```